### DISCUSSION TITLE
EOS Install 7 Error

### DISCUSSION THREAD

**Discussion by doobes** [elementary/triage#168] [Q&A]
Whelp, 

I've got a Dell Inspiron 7437 that runs EOS v6.1 just peachy.

Doing a complete disk wipe and the attempting to install EOS 7, it tells me the install failed "possibly" due to a hardware failure. 

I've been through this numerous times, and redownloaded the ISO and made boot USB's using both Linux and Windows with the same result.

Some snippets from the "Details window"

INFO: Extracting /cdrom/casper/filesystem.squashfs – 
DEBUG: "unsquashfs" "-f" "-d" "/tmp/distinst.4Xc09V1KFW00" "/cdrom/casper/filesystem.squashfs" – 
ERROR: extracting error: archive extraction failed with a=status: exit status 1 –

On another, but related topic, I've never had the erase everything and install selection succeed. I've always had to boot a gparted disk and clean everything off the target disk to get EOS to install.

I tried that on this laptop to no avail.

V7 works just fine in demo mode though :(

**Comment by doobes** [comment]
Tried it again without wiping the disk and got a bunch of disk type errors that I'm not inclined to retype.

**Reply by garethstewart** [reply]
I've just had the same on an old MacBook that was running 6.1 ... install failed, and now it can't find an OS to boot. *sigh*

**Comment by doobes** [comment]
Sorry.

I went back to v6.1.

Maybe I'll try v7 when it's done.....
